client/pages/index.jsx

```
@@ -1,3 +1,4 @@
+/* eslint-disable react/no-unescaped-entities */
```

It's better not to ignore this error. I think you should properly escape the single quotes.

---

hi @hitochan777 sorry, are you talking about escaping the single quotes in the package.json file? I've done that. 

The error is on lines 12 and 15 of index.jsx, and when I follow the docs (https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/no-unescaped-entities.md) it seems to be suggesting that I haven't closed the <button> tag properly?

---

meant to say the button tag

---

it's ok @hitochan777 , realised what you mean and have figured it out!

---

Sorry for the confusion!